### DISCUSSION TITLE
DeepMD-kit install

### DISCUSSION THREAD

**Discussion by man710** [deepmodeling/deepmd-kit#2642] [Q&A]
For the installation of DeepMD-kit so that I can use it for the deepmd pairstyle is this the sequence I have to follow from the instruction manual?

1. Download the source code
2.) 1.4 Install Python interface with pip 
      pip install deepmd-kit[cpu]
   pip install deepmd-kit[gpu,cu11]
3.) Install the Tensorflow's python interface using virtual environment
   virtualenv -p python3 $tensorflow_venv
   source $tensorflow_venv/bin/activate
   pip install --upgrade pip 
   pip install --upgrade tensorflow==2.3.0
4.) Install the DeepMD-kit's C++ interface 
   cd /some/workspace
git clone --recursive https://github.com/deepmodeling/deepmd-kit.git deepmd-kit
    
  cd deepmd-kit
deepmd_source_dir=`pwd`
cd $deepmd_source_dir/source
mkdir build 
cd build
cmake -DUSE_TF_PYTHON_LIBS=TRUE DLAMMPS_VERSION_NUMBER=<value> -DCMAKE_INSTALL_PREFIX=$deepmd_root                                                                                         -DCMAKE_INSTALL_PREFIX=$deepmd_root 
make 
make install

5.) Install LAMMPS's DeePMD-kit module
cd $deepmd_source_dir/source/build
make lammps
cd /some/workspace
tar xf lammps-stable.tar.gz
cd lammps-31Mar17/src/
cp -r $deepmd_source_dir/source/build/USER-DEEPMD .
make yes-user-deepmd
make mpi -j4

Apart from this do I need to follow any other steps for example 2.2.1, 3.4.3 of the manual.
I will be very grateful for the help provided.

**Comment by njzjz** [comment]
Please provide the manual you are referring to. I don't find 3.4.3 in the current manual.
